src/components/AttachmentPicker/index.native.js

```diff
@@ -79,13 +94,18 @@ class AttachmentPicker extends Component {
                 text: this.props.translate('attachmentPicker.chooseFromGallery'),
                 pickAttachment: () => this.showImagePicker(launchImageLibrary),
             },
-            {
-                icon: Paperclip,
-                text: this.props.translate('attachmentPicker.chooseDocument'),
-                pickAttachment: () => this.showDocumentPicker(),
-            },
         ];
 
+        if (this.props.type !== CONST.ATTACHMENT_PICKER_TYPE.IMAGE) {
```

Comment:
This is probably worth adding a comment to explain why this logic is being done

Reply:
Sure!

Reply:
`// Excluding choose document option for picker type - image`

Reply:
The best strategy for writing a comment is to describe the "why" and this comment is only describing "what". A better comment could be:

`// When selecting an image on a native device, it would be redundant to have a second option for choosing a document, so it is excluded in this case.`

Reply:
Thanks for your guidance!